FLEDGE.md

```diff
@@ -179,9 +179,12 @@ included within the URL. Note that the lifetime of an Interest Group is not affe
 
 The updates are done after auctions so as not to slow down
 the auctions themselves.  The updates are rate limited to running at most daily to
-conserve resources, and timeout after 30 seconds.  An update request only contains information from the single site
-where the user was added to the interest group.  At a later date we can consider
-potential side channel mitigations (e.g.
+conserve resources, and timeout after 30 seconds; however, the `updateIfOlderThanMs`
+field of the [trusted bidding signals response](#31-fetching-real-time-data-from-a-trusted-server)
+maybe be used to trigger a single post-auction update that will run even if the last
```

Comment:
The post-auction updating what we do now, but may not be the long-term solution.  Let's just describe it as happening soon.  You don't have to use my suggested wording.
```suggestion
can be used to trigger a more timely, than daily, update that will run even if the last
```

Reply:
Done -- I still kept the word "single" to show that this new timing doesn't apply for all future updates, just the next one.